pkgs/clightning-plugins/default.nix

```diff
@@ -0,0 +1,61 @@
+pkgs: nbPython3Packages:
+
+let
+  inherit (pkgs) lib;
+
+  src = pkgs.fetchFromGitHub {
+    owner = "lightningd";
+    repo = "plugins";
+    rev = "6cd472636926f05a9c472139fabe1ff11c90aa6a";
+    sha256 = "13cccwxanhmwmhr4prs13gn5am9l2xqckk80ad41kp2a4jq4gdsy";
```

Comment:
This should be `1lisx85vzsfzjhdc6zdz0l6bcrdgg6rp5xbc5jmx93mv8qqg2cns`. For some reason it still deployed on my end with this and didn't copy over the new script. Seems like the `src` doesn't update properly until you change this.

Reply:
Dammit.  I pasted from the wrong buffer!  When I did a test deployment, it succeeded like in your case and I even saw new paths with the suffix `6cd4726` both in the deployment output and in the clightning config.  This seems like a problem.  Builds should always fail if the integrity check fails.

Reply:
The store path of a fixed-output derivation is only determined by its name and the `sha256` attr. The other attrs are only used when building the drv. To trigger a rebuild (and a new integrity check) you have to change the `sha256`.

Reply:
Ok that makes sense.  In light of this behavior, how about we make use the first 7 digits of the sha256 for the version instead of the first digits of the rev?

Reply:
If a version number is unavailable, `version` should consist of a maximally unambiguous and easily identifiable fact about the source. In nixpkgs, it's common practice to use the src's git rev. See:
```
cd $(nix eval '(<nixpkgs>)'); nix run nixpkgs.ripgrep -c rg "version = .*substring"
```

The src `sha256` fails at the "easily identifiable" part and its value is already implicitly used to calculate the plugin drv hash (`/nix/store/{HASH}-...`).